__tests__/api/templates.test.ts

```diff
@@ -5,7 +5,8 @@
 /* eslint-disable  @typescript-eslint/no-explicit-any */
 import { createMocks, RequestMethod } from "node-mocks-http";
 import Redis from "ioredis-mock";
-import templates from "@pages/api/templates";
+import templates from "@pages/api/templates/[...formID]";
+import { default as templatesRoot } from "@pages/api/templates/index";
```

Comment:
For readability we could use the same pattern as the other default imports with aliases:  `import templatesRoot from "@pages/api/templates/index"`